test/e2e_node/swap_test.go

```diff
@@ -19,6 +19,7 @@ package e2enode
 import (
 	"context"
 	"fmt"
+	"k8s.io/kubernetes/pkg/kubelet/util/swap"
```

Comment:
this is a super nit but better if you move it under the other imports, I also don't know if the linter will protest here